src/App.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import Todo from '../src/component/Todo';
+import './App.css';
+
+class App extends Component {
```

Comment:
### Critical

If this component isn't going to have anything to do with state, it's safe to just turn it to a presentation or functional component. This helps reduce bundle size as class components ship with some extra complexities.